src/rpc/RPC.cpp

```diff
@@ -191,6 +191,7 @@ static HandlerTable handlerTable{
     {"ledger", &doLedger, {}},
     {"ledger_data", &doLedgerData, LimitRange{1, 100, 2048}},
     {"nft_buy_offers", &doNFTBuyOffers, LimitRange{1, 50, 100}},
+    {"nft_info", &doNFTInfo},
```

Comment:
Does this only add `nft_info`?

Wasn't the plan to do nft_transactions as well? Else, why are you adding NFTTransactions to the database?